telegram-bot/main.go

```diff
@@ -29,6 +30,29 @@ func ButtonLinks(ID int64, ButtonText string, ButtonUrl string, MessageText stri
 	bot.Send(msg)
 }
 
+func tweet() {
```

Comment:
Add comments at the beginning of the function to document what this function is about.

Reply:
Please check and see.